### PR TITLE
AMB-325 add smoke tests by running the "Init Tests Suite" and the "Ha…

### PR DESCRIPTION
**Pull request by artronics**
…ppy Path Tests" folders

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/templates/smoke-tests.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+              export BASE_URL="${{ parameters.environment }}"
+              export ROOT_PATH="$(SERVICE_BASE_PATH)"
+              export CLIENT_ID="$(CLIENT_ID)"
+              export JWT_RS256_PRIVATE_KEY=$(echo "$(JWT_RS256_PRIVATE_KEY)" | base64 -d | sed -z 's/\n/\\n/g')
+              export JWT_RS512_PRIVATE_KEY=$(echo "$(JWT_RS512_PRIVATE_KEY)" | base64 -d | sed -z 's/\n/\\n/g')
+
+              envsubst < $test_dir/environments/internal-dev.postman.json > $test_dir/environments/prefilled.internal-dev.postman.json
+
+              echo docker run -v ${test_dir}:/etc/newman postman/newman run ambulance_analytics.json \
```
Comment (JalfResi) [inline azure/templates/smoke-tests.yml:27]: Remove this - this task is for smoke tests only

File: azure/azure-pr-pipeline.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ extends:
     apigee_deployments:
       - environment: internal-dev
         post_deploy:
-          - template: templates/tests.yml
```
Comment (JalfResi) [inline azure/azure-pr-pipeline.yml:41]: Add the tests back in - we want to run the integration tests AND the smoke test, each as a separate step

Comment (JalfResi) [reply azure/azure-pr-pipeline.yml:41]: And we want the smoke tests to run on INT and PROD envs (PROD only has smoke tests, not integration tests)

Comment (artronics) [reply azure/azure-pr-pipeline.yml:41]: @JalfResi I added this to internal-dev for testing. I don't think we can add two templates inside the `post_deploy` step. I made some changes to find a work around by I couldn't.

**Review by JalfResi — APPROVED** [review]

**Pull request merged**